electrum/gui/qt/three_keys_windows.py

```diff
@@ -1,30 +1,300 @@
-from .util import read_QIcon
-from .main_window import ElectrumWindow
+import enum
+
+from PyQt5.QtCore import Qt, QRect
+from PyQt5.QtGui import QStandardItem
+from PyQt5.QtWidgets import QVBoxLayout, QLabel, QWidget, QHBoxLayout, QHeaderView, QStyleOptionButton, QStyle, \
+    QGridLayout, QCompleter, QComboBox, \
+    QStyledItemDelegate
+
 from electrum.i18n import _
-from .recovery_list import RecoveryTab
+from .amountedit import BTCAmountEdit, MyLineEdit, AmountEdit
+from .completion_text_edit import CompletionTextEdit
+from .confirm_tx_dialog import ConfirmTxDialog
+from .main_window import ElectrumWindow
+from .recovery_list import RecoveryTabARStandalone, RecoveryTabAIRStandalone
+from .util import read_QIcon, HelpLabel, EnterButton
+from ...mnemonic import load_wordlist
+from ...plugin import run_hook
+from ...three_keys import short_mnemonic
+
+
+class CheckableHeader(QHeaderView):
+    def __init__(self, orientation, parent=None):
+        super().__init__(orientation, parent)
+        self.parent = parent
+        self.is_on = False
+
+    def paintSection(self, painter: 'QPainter', rect: 'QRect', logical_index: int):
+        painter.save()
+        super().paintSection(painter, rect, logical_index)
+        painter.restore()
 
+        # assure set only in first column
+        if logical_index == 0:
+            option = QStyleOptionButton()
+            option.rect = QRect(
+                23, 3,
+                14, 14
+            )
+            if self.is_on:
+                option.state = QStyle.State_On
+            else:
+                option.state = QStyle.State_Off
+            self.style().drawPrimitive(QStyle.PE_IndicatorCheckBox, option, painter)
+
+    def mousePressEvent(self, event):
+        if self.is_on:
+            self.is_on = False
+        else:
+            self.is_on = True
+        super().updateSection(0)
+        super().mousePressEvent(event)
+
+
+class TableItem(QStandardItem):
+    def __init__(self, text, if_checkable=False):
+        super().__init__(text)
+        self.setCheckable(if_checkable)
+        self.setTextAlignment(Qt.AlignRight)
+
+
+class ElectrumMultikeyWalletWindow(ElectrumWindow):
+    LABELS = ['Date', 'Confirmation', 'Balance']
 
-class ElectrumARWindow(ElectrumWindow):
     def __init__(self, gui_object: 'ElectrumGui', wallet: 'Abstract_Wallet'):
         super().__init__(gui_object=gui_object, wallet=wallet)
+        self.alert_transactions = []
         self.recovery_tab = self.create_recovery_tab(wallet, self.config)
         # todo add proper icon
         self.tabs.addTab(self.recovery_tab, read_QIcon('recovery.png'), _('Recovery'))
 
     def create_recovery_tab(self, wallet: 'Abstract_Wallet', config):
-        return RecoveryTab(self, wallet, config)
+        raise Exception("Base class method should not be implemented")
```

Comment:
`NotImplementedError` seems to be more suitable in this situation

Reply:
Done